pkg/webui/console/views/admin-packet-broker/messages.js

```diff
@@ -18,19 +18,30 @@ export default defineMessages({
   packetBrokerInfoText:
     'Packet Broker can be used to exchange traffic with other LoRaWAN networks to share coverage and improve the overall network performance.',
   packetBrokerWebsite: 'Packet Broker website',
+  registrationStatus: 'Registration status',
   registerNetwork: 'Register network',
-  registerThisNetwork: 'Register this network',
-  network: 'Network: {network}',
-  homeNetworkEnabled: 'Home network enabled',
-  homeNetworkDisabled: 'Home network disabled',
-  forwarderEnabled: 'Forwarder enabled',
-  forwarderDisabled: 'Forwarder disabled',
+  networkVisibility: 'Network visibility',
   packetBrokerRegistrationDesc:
-    "To enable package exchange from or to your home network, it is necessary to register your network. This will make your network known to the Packet Broker and enable you to configure your network's package exchange behavior.",
+    "To enable package exchange from or to your home network, it is necessary to register your network. This will make your network known to Packet Broker and enable you to configure your network's packet exchange behavior.",
```

Comment:
We're mixing terms:

- Exchange traffic
- Peering
- Package exchange

These are all interchangeable.

To add more terms, we use the terms messages (TTS), frames (used in LoRaWAN spec, lower level), packets (like Packet Broker) and packages.

I think we should start stripping a few terms here, beginning with "package" and "package exchange". Instead, we should go with "peering", "traffic exchange" and "messages".

Reply:
Sorry, I this was an unintentional change actually.